src/features/embalm/stepNavigator/components/StepStatusIndicator.tsx

```diff
@@ -15,8 +16,8 @@ export function StepStatusIndicator({ status, index }: StepStatusIndicatorProps)
         h={statusSize}
         w={statusSize}
         border="1px solid"
-        borderColor="brand.950"
-        backgroundColor="brand.950"
+        borderColor={isDisabled ? 'brand.400' : 'brand.950'}
```

Comment:
do you think we should make a named "disabled" in the theme we could use instead of remembering it's 400 (or maybe it's 300)

Reply:
Yes, this is a good idea.